containers/provision/centos:8.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+dnf install -y 'dnf-command(config-manager)' \
+	       'dnf-command(builddep)' \
+	       make \
+	       rpm-build \
+	       createrepo_c \
+	       valgrind \
+	       git-daemon \
+	       python3-flask \
+	       python3-requests \
+	       python3-pytest \
+	       python3-six \
+	       procps-ng
+
+dnf config-manager -y --enable PowerTools
```

Comment:
`PowerTools` == `AppStream` ?

Reply:
Ne, AppStream is already enable in the image.
```
Name         : libarchive-devel
Version      : 3.3.2
Release      : 8.el8_1
Architecture : x86_64
Size         : 131 k
Source       : libarchive-3.3.2-8.el8_1.src.rpm
Repository   : PowerTools
```
```
Name         : json-c-devel
Version      : 0.13.1
Release      : 0.2.el8
Architecture : x86_64
Size         : 44 k
Source       : json-c-0.13.1-0.2.el8.src.rpm
Repository   : PowerTools
```

Reply:
CRB then I guess..